src/entrypoints/background.ts

```
@@ -0,0 +1,26 @@
+
+export default defineBackground({
+  type: 'module',
+  main() {
+    const storage = new Storage();
+    storage.initAsync();
+    const operationResolver = new OperationResolver(storage);
+
+    chrome.runtime.onMessage.addListener(async function (message: Message) {
+      switch (message.type) {
+        case 'gestures':
+          const context = new Context(message.gestures, message.selectedText, message.linkUrl);
```

Biome (2.5.3)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)